src/pages/Admin/Venue/AnnouncementStatus/AnnouncementStatus.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { faCheck, faTimes } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
+
+import { Banner } from "types/banner";
+
+import { isDefined } from "utils/types";
+
+import "./AnnouncementStatus.scss";
+
+export interface AnnouncementStatusProps {
+  banner?: Banner;
+}
+
+const ENABLED__TEXT = "enabled";
+const DISABLED__TEXT = "disabled";
+
+export const AnnouncementStatus: React.FC<AnnouncementStatusProps> = ({
+  banner,
+}) => {
+  const fullscreenImg = banner?.isFullScreen ? faCheck : faTimes;
+  const closeImg = banner?.isForceFunnel ? faCheck : faTimes;
+
+  const fullscreenText = banner?.isFullScreen ? ENABLED__TEXT : DISABLED__TEXT;
+  const forceText = banner?.isForceFunnel ? ENABLED__TEXT : DISABLED__TEXT;
+
+  const urlText = banner?.isActionButton ? banner?.buttonUrl : "blank";
+
+  const urlTextClasses = classNames("AnnouncementStatus__url-text", {
+    "AnnouncementStatus__url-text--empty": !banner?.isActionButton,
+  });
+
+  if (!isDefined(banner?.content)) return null;
```

Comment:
is it unexpected behavior? can we add a message to the console that the banner is missing `content` and that's why we can't render the content?

or what is it for?

Reply:
it's required to hide the additional fields in case announcement is empty 
![image](https://user-images.githubusercontent.com/56254806/133979523-3835ee5e-04e9-4fc0-982b-9179beedf621.png)
![image](https://user-images.githubusercontent.com/56254806/133979538-00663ece-8e42-4995-845a-ce292cb80f53.png)